dev/storage.h

```diff
@@ -999,9 +984,9 @@ namespace sqlite_orm {
             std::map<std::string, sync_schema_result> sync_schema(bool preserve = false) {
```

Comment:
@fnc12 Now that it is possible to iterate specific schema objects (filter), shall we create tables first, then all other schema objects? This would make it possible to specify indexes and triggers anywhere in the storage definition.